src/components/NavBar/NavBar.jsx

```diff
@@ -47,30 +51,34 @@ export const NavBar = () => {
 					</div>
 				) : (
 					<div className="flex-column">
-						<Link to="/login">
+						<Link to="/login" className="flex-column ">
 							<i class="fas fa-user btn-icon"></i>
+							<span className="text-xxs pointer">Login</span>
 						</Link>
-						<span className="text-xxs pointer">Login</span>
 					</div>
 				)}
 
-				<div>
+				<div className="flex-column">
 					<Link to="/wishlist">
-						<div className="badge-wrapper flex-column">
+						<div className="badge-wrapper ">
 							<i class="fas fa-heart btn-icon"></i>
 
-							<span className="badge badge-top badge-s badge-red">11</span>
+							<span className="badge badge-top badge-s badge-red">
+								{wishlist.wishlistProducts.length}
```

Comment:
One small suggestion is that you could destructure above and just display it here. This could make your code more readable.